src/index.ts

```diff
@@ -1,19 +1,42 @@
 "use client";
 
 export { ProseMirror } from "./components/ProseMirror.js";
-export { EditorProvider } from "./contexts/EditorContext.js";
-export { LayoutGroup, useLayoutGroupEffect } from "./contexts/LayoutGroup.js";
+
+export { Editor } from "./components/Editor.js";
+export type { EditorProps } from "./components/Editor.js";
+
+export { EditorContext } from "./components/EditorContext.js";
```

Comment:
Why don't I merge https://github.com/nytimes/react-prosemirror/pull/100 into this PR and also remove `<Editor />` from there along with `EditorContext`?

Reply:
Yup, good with that!

Reply:
Should I remove `NodeViewsContext` and `NodeViewsContextValue` as well? I think probably.

Reply:
Agreed

Reply:
I'll try not even exporting `EditorProps`. I hope that'll make TypeScript just reflect all the props as individual props without suggesting that users should ever type something as `EditorProps` or pass around an `EditorProps`. They're just the props you pass to `<ProseMirror />`.

Reply:
I think I'd rather leave that exported. I see what you mean, but I really prefer when there are public types for public interfaces. It would make it much easier to type a wrapper component, for example. I find myself constantly doing annoying `Parameters<typeof someFunc>[0]` nonsense when they're not available.

Reply:
Hmmm. I merged that in, but let me think about it. Maybe I can export `ProseMirrorProps`?

Reply:
Yeah I think that's actually what I want! I mean I know they're the same type, but what I actually want is for the props type of the ProseMirror component to be public